R/mod_export_data.R

```diff
@@ -115,36 +120,36 @@ mod_export_data_ui <- function(id) {
 
 mod_export_data_server <- function(input, output, session, conn) {
   ns <- session$ns
-  
```

Comment:
See what can be done about these spaces, we do not want these changed as it adds unnecessary checking steps in the code review process. Try to avoid any indentation changes as well.